creativecloud/features/changeBg/changeBg.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable no-restricted-syntax */
+let excelLink = '';
+const configObj = {};
+const customElem = document.createElement('ft-changebackgroundmarquee');
```

Comment:
If this is only required inside changebg function we do not need to have it as a global variable. You can keep it as a const inside the function only.